stdlib/types.pyi

```diff
@@ -366,6 +388,7 @@ if sys.version_info >= (3, 10):
     from builtins import _NotImplementedType
 
     NotImplementedType = _NotImplementedType  # noqa F811 from builtins
+    @final
     class Union:
```

Comment:
This ended up getting renamed to `UnionType`: https://github.com/python/cpython/blob/3.10/Lib/types.py#L301. Could you change that while you're at it?

Reply:
No problem; done as of https://github.com/python/typeshed/pull/5837/commits/141ca5eda375710090b80bd1583a82eff3569e4b.